.vscode/settings.json

```
@@ -0,0 +1,8 @@
+{
+    "editor.detectIndentation": false,
+    "editor.tabSize": 4,
+    "editor.insertSpaces": true,
+    "[csharp]": {
+        "editor.rulers": [140]
+    },
+}
```

Looks like this needs a newline in the end of file?
```suggestion
}

```